Makefile

```diff
@@ -188,9 +238,9 @@ endif
 install-engine:
 	@-echo "Installing OpenRA engine to $(DATA_INSTALL_DIR)"
 	@$(INSTALL_DIR) "$(DATA_INSTALL_DIR)"
-	@$(INSTALL_PROGRAM) OpenRA.Game.exe "$(DATA_INSTALL_DIR)"
-	@$(INSTALL_PROGRAM) OpenRA.Server.exe "$(DATA_INSTALL_DIR)"
-	@$(INSTALL_PROGRAM) OpenRA.Utility.exe "$(DATA_INSTALL_DIR)"
+	@$(INSTALL_PROGRAM) OpenRA$(EXT) "$(DATA_INSTALL_DIR)"
```

Comment:
These will always be `.dll`! Mono doesn't give an exe extension when you compile an exectutable against netstandard. This may be the result of undefined behaviour, but its probably not going to change given the trajectory towards merging mono and core.